frontend/src/lib/lemon-ui/LemonSkeleton/LemonSkeleton.tsx

```diff
@@ -5,15 +5,29 @@ import './LemonSkeleton.scss'
 
 export interface LemonSkeletonProps {
     className?: string
+    /** Height of the skeleton bar */
+    height?: number
     /** Repeat this component this many of times */
     repeat?: number
     /** Used in combination with repeat to progressively fade out the repeated skeletons */
     fade?: boolean
     active?: boolean
 }
-export function LemonSkeleton({ className, repeat, active = true, fade = false }: LemonSkeletonProps): JSX.Element {
+export function LemonSkeleton({
+    className,
+    repeat,
+    height = 4,
```

Comment:
This will be a bit confusing, because it means we'll be setting width with `className`, and height with `height`… And we already _try_ to use `className` for this in a bunch of places (which I think doesn't work), e.g. [here](https://github.com/PostHog/posthog/blob/b19e7df74a5250d3bee3f4f56f371205658b67b2/frontend/src/scenes/notebooks/Nodes/NotebookNodeSurvey.tsx#L55). So it'd be much safer to put `h-4` in the `className` default and update the existing call sites overriding this to have a height class.